jujugui/static/gui/src/app/components/deployment-flow/expert-budget/expert-budget.js

```diff
@@ -73,6 +73,7 @@ class DeploymentExpertBudget extends React.Component {
 
 DeploymentExpertBudget.propTypes = {
   budget: PropTypes.any,
+  estimateWithSLA: PropTypes.any,
```

Comment:
any? Why? Please add a comment.

Reply:
I didn't want to limit it to be a number or a string so this way it can be either depending on where the component is used. I'll add a comment